tests/schemas/valid/test_placeholder_default_value.json

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
I don't think we need an entire new schema. Can we not add two new questions into an existing one?

Reply:
Ok added percentage and unit transforms showcase questions and interstitials to existing placeholder transforms schema, thanks!